src/mudlet-lua/lua/LuaGlobal.lua

```diff
@@ -146,25 +154,64 @@ if getMudletLuaDefaultPaths then
   end
 end
 
+if package.config:sub(1,1) == '\\' then
+  -- this is a windows platform, make all the path separators be '\\'
+  for i, path in ipairs(prefixes) do
+    prefixes[i] = string.gsub(path, '/', '\\')
+    if debugLoading then
+      echo("Directory separator conversion: \"" .. path .. "\" becomes: \"" .. prefixes[i] .. "\"\n")
+    end
+  end
+else
+  -- this is a non-windows platform make all the path separators be '/'
+  for i, path in ipairs(prefixes) do
+    prefixes[i] = string.gsub(path, '\\', '/')
+    if debugLoading then
+      echo("Directory separator conversion: \"" .. path .. "\" becomes: \"" .. prefixes[i] .. "\"\n")
+    end
+  end
+end
+
+if debugLoading then
+  echo("Current directory is: \"" .. lfs.currentdir() .. "\".\n")
+end
+
 local prefix
 for i = 1, #prefixes do
-  if lfs.attributes(prefixes[i]) then
+  -- lfs.attributes returns a table if the given file-system object exists
+  if debugLoading then
+    echo("Testing: \"" .. prefixes[i] .. "LuaGlobal.lua\"...")
+  end
+  if lfs.attributes(prefixes[i] .. "LuaGlobal.lua") then
+    if debugLoading then
+      echo(" found something!\n")
+    end
     prefix = prefixes[i]
     break
+  else
+    if debugLoading then
+      echo(" not found.\n")
+    end
   end
 end
 
--- For some reason on windows, mudlet-lua/lua/ does not register as a directory with the above strategy.
--- Thus, if chosen we need to append a slash.
-if prefix == "mudlet-lua/lua" then
-  prefix = prefix .. "/"
+-- For some reason on windows, mudlet-lua/lua/ does not register as a directory
+-- with the above strategy. Thus, if chosen we need to append a slash.
+-- Now changed to \ to reflect need to accomodate windows backwardness in
+-- regard to specifying paths and files...
+if prefix == "mudlet-lua\\lua" then
```

Comment:
Are you sure you run into this if-clause? It looks mightily strange...

Reply:
Well, it originally was `mudlet/lua` but I am not sure when it would be hit so I thought it best to leave it...

Reply:
All prefixes now end with a slash, so you should not run into this anymore.

Reply:
OK, I'll remove this line and see how we get on...

Reply:
Have nuked it...